docs/blog/001-train-custom-dataset/README.md

```diff
@@ -0,0 +1,245 @@
+
+To run a sweep, you can just call,
+
+```
+python tools/hpo/wandb_sweep.py --model padim --config ./path_to_config.yaml --sweep_config tools/hpo/sweep.yaml"
+```
+
+In case `model_config` is not provided, the script looks at the default config location for that model. Note, you will need to have logged into a wandb account to use HPO search and view the results.
+
+A sample run is visible in the screenshot below.
+<div align="center">
+<img src="docs/blog/../../images/hpo.gif">
+</div>
+
+## Benchmarking
+To add to the suit of experiment tracking and optimization, anomalib also includes a benchmarking script for gathering results across different combinations of models, their parameters, and dataset categories. The model performance and throughputs are logged into a csv file that can also serve as a means to track model drift. Optionally, these same results can be logged to Weights and Biases and TensorBoard. A sample configuration file is shown in the screenshot below.
+
+```yaml
+seed: 42
+compute_openvino: false
+hardware:
+  - cpu
+  - gpu
+writer:
+  - wandb
+  - tensorboard
+grid_search:
+  dataset:
+    category:
+      - colour
+      - crack
+    image_size: [128, 256]
+  model_name:
+    - padim
+    - stfpm
+```
+
+This configuration computes the throughput and performance metrics for CPU and GPU for two categories of the toy dataset for Padim and STFPM models. The dataset can be configured in the respective model configuration files. By default, `compute_openvino` is set to False to support instances where OpenVINO requirements are not installed in the environment. Once installed, this flag can be set to True to get throughput on OpenVINO optimized models. The writer parameter is optional and can be set to `writer: []` in case the user only requires a csv file without logging to TensorBoard or Weights and Biases. It is also a good practice to set a value of seed to ensure reproducibility across runs and thus, is set to a non-zero value by default.
+
+Once a configuration is decided, benchmarking can easily be performed by calling
+
+```bash
+python tools/benchmarking/benchmark.py --config tools/benchmarking/benchmark_params.yaml
+```
+
+A nice feature about the provided benchmarking script is that if the host system has multiple GPUs, the runs are parallelized over all the available GPUs for faster collection of result.
+
+**Call to Action**
+
+The Anomalib repository is actively maintained by some of Intel’s top researchers. Their goal is to provide the AI-community with top of the class performance and accuracy while also providing a great user-experience for developers. Checkout out the repo and install anomalib today!
```

Comment:
Same here. "top researchers" may come across a bit overly confident if we are the ones writing this about ourselves (I mean, it's true of course, but we don't want to be arrogant).

Reply:
Addressed